test/options/rule-delimiter-scss/rule-delimiter.expected.scss

```diff
@@ -0,0 +1,19 @@
+.foo {
+  @import border-radius(5px);
+  background: red;
+
+.bar {
+
```

Comment:
So this is funny, when I run in CLI it does do this, however the tests output the above. Maybe some other rule is handling this or something?